src/rules/monitors.ts

```diff
@@ -151,8 +156,18 @@ monitors:
 monitors:
   memory:
     - swarmstash
-swarm:
-  minimum_node_count: "1"
+---
+# kind: scheduler-swarm
+
+version: '3.3'
+services:
+  mysql:
+    image: mysql
+---
+# kind: scheduler-swarm
+
+#this is not valid yaml
+{
```

Comment:
But before anything else, I need to see if there's a way to fix this sort of thing
(by ignoring mesg-yaml-valid from the 2nd document on, for example)